src/Shared/runtime/Http2/Hpack/HPackEncoder.cs

```diff
@@ -199,6 +199,42 @@ public static bool EncodeIndexedHeaderField(int index, Span<byte> destination, o
             return false;
         }
 
+        /// <summary>Encodes the status code of a response to the :status field.</summary>
+        public static bool EncodeIndexedStatusHeader(int statusCode, Span<byte> destination, out int bytesWritten)
```

Comment:
```suggestion
        public static bool EncodeStatusHeader(int statusCode, Span<byte> destination, out int bytesWritten)
```
Indexing or not is up to the implementation.